components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -2612,6 +2612,19 @@ public API createNewAPIVersion(String existingApiId, String newVersion, Boolean
             saveGraphqlSchemaDefinition(newAPIId, graphQLSchema, organization);
         }
 
+        // Copy labels
+        try {
+            List<Label> labels = getAllLabelsOfApi(existingApiId);
+            if (labels != null && !labels.isEmpty()) {
+                log.info("Copying " + labels.size() + " labels from API: " + existingApiId + " to new API version: "
```

Comment:
Remove this info log and add a debug log instead. Otherwise it will print unnecessary logs in the log file. When adding the debug log, please use `if (log.isDebugEnabled())` check as well.